Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -569,23 +569,18 @@ In addition:
+- **Custom or named pasteboards (`UIPasteboard(name:create:)`, `UIPasteboard.withUniqueName()`)**: App-specific or team-specific. Non-persistent by default since iOS 10 and restricted to the creating app or apps from the same team ID. Apple discourages using persistent custom pasteboards and recommends [using App Groups](https://developer.apple.com/documentation/Xcode/configuring-app-groups) for sharing data between apps of the same developer.
 
-The [`UIPasteboard`](https://developer.apple.com/documentation/uikit/uipasteboard "UIPasteboard") enables sharing data within an app, and from an app to other apps. There are two kinds of pasteboards:
+In earlier iOS versions, malicious apps could monitor the pasteboard from the background.
 
-- **systemwide general pasteboard**: for sharing data with any app. Persistent by default across device restarts and app uninstalls (since iOS 10).
-- **custom / named pasteboards**: for sharing data with another app (having the same team ID as the app to share from) or with the app itself (they are only available in the process that creates them). Non-persistent by default (since iOS 10), that is, they exist only until the owning (creating) app quits.
-
-**Security Considerations:**
-
-- Users cannot grant or deny permission for apps to read the pasteboard.
-- Since iOS 9, apps [cannot access the pasteboard while in background](https://forums.developer.apple.com/thread/13760 "UIPasteboard returning null from Today extension"), this mitigates background pasteboard monitoring. However, if the _malicious_ app is brought to foreground again and the data remains in the pasteboard, it will be able to retrieve it programmatically without the knowledge nor the consent of the user.
-- [Apple warns about persistent named pasteboards](https://developer.apple.com/documentation/uikit/uipasteboard?language=objc "Pasteboard Security and Privacy Changes in iOS 10") and discourages their use. Instead, shared containers should be used.
-- Starting in iOS 10 there is a new Handoff feature called Universal Clipboard that is enabled by default. It allows the general pasteboard contents to automatically transfer between devices. This feature can be disabled if the developer chooses to do so and it is also possible to set an expiration time and date for copied data.
+- Since iOS 9, access to the pasteboard is restricted to apps running in the foreground, significantly reducing the risk of passive clipboard sniffing. However, if sensitive data remains in the pasteboard and a malicious app is later brought to the foreground (or an app widget which is on the fireground everytime the user is on the screen it's located), it can still access that data without user consent or visibility. See [example attack](https://www.thedailybeast.com/facebook-is-spying-on-your-clipboard).
```

Comment:
```suggestion
- Since iOS 9, access to the pasteboard is restricted to apps running in the foreground, significantly reducing the risk of passive clipboard sniffing. However, if sensitive data remains in the pasteboard and a malicious app is later brought to the foreground (or an app widget which is on the fireground every time the user is on the screen it's located), it can still access that data without user consent or visibility. See [example attack](https://www.thedailybeast.com/facebook-is-spying-on-your-clipboard).
```